src/pages/grocery-list.js

```diff
@@ -97,10 +129,24 @@ const GroceryListPage = () => {
             {showError && <p className="sage-list--error">Please enter a valid email</p>}
             <Button className="sage-list--send-button" onClick={handleClickSend}>Send</Button>
           </div>
-        ): (
-          <p className="sage-list--sent">Your grocery list has been sent! Please check your inbox.</p>
-        )}
+        ) : (
+            <p className="sage-list--sent">Your grocery list has been sent! Please check your inbox.</p>
+          )}
       </Modal>
+      
+      <Modal
+          size="small"
+          warning={true}
+          isOpen={showCloseWarning}
+          onClose={() => setShowClosedWarning(false)}
+          title="Are you sure?">
+          Your changes will be lost. Are you sure you want to exit?
```

Comment:
The wording here and in the button should say something about the list not being sent with the new changes, not exit stuff